apis/v1alpha1/types.go

```diff
@@ -28,6 +28,8 @@ type ProviderConfigSpec struct {
 	Credentials ProviderCredentials `json:"credentials"`
 	// AccountID required by nerdgraph API
 	AccountID string `json:"account_id,omitempty"`
+	// Region of the account (default: US)
```

Comment:
Would it make sense to add a validation for this since there are only 2 allows values?
https://docs.newrelic.com/docs/accounts/accounts-billing/account-setup/choose-your-data-center/

```suggestion
	// Region of the account (default: US)
	// +kubebuilder:validation:Enum=US;EU
```